### PR TITLE
Support multiple collections for add function

### PR DESCRIPTION
**Pull request by mohammadKhalafi**
issue : https://github.com/mavoweb/mavo/issues/829

### CODE REVIEW

**Review by LeaVerou — CHANGES_REQUESTED** [review]
Hi there, thank you so much! Maybe it's that I've just woken up, but i found it hard to follow the logic. Some comments would help, especially around L106-113

**Comment by mohammadKhalafi** [comment]
> Hi there, thank you so much! Maybe it's that I've just woken up, but i found it hard to follow the logic. Some comments would help, especially around L106-113

Hi! a few comments is added.

**Comment by LeaVerou** [comment]
Thanks! At a quick glance it looks alright, though I haven't tested it or done a deep dive.

@DmitrySharabin could you take a look as well? Note this question I raised in the original issue:

> One question is: What do we do when some items in an array belong to collections and others do not? Do you add to those collections, or do you give up altogether?

**Review by lewisushindi — APPROVED** [review]

**Comment by DmitrySharabin** [comment]
Hey there,

Sorry for the delay. I played a bit with the suggested changes. And it looks like the generated `Add <collection>` buttons (framed in the screenshot) stopped working.

image

@mohammadKhalafi You may find these tests useful for debugging: https://test.mavo.io/actions.html#add. And could you please provide a test case on Codepen so it is obvious how the new feature will work?

**Comment by LeaVerou** [comment]
Closing due to no activity for over a year.
